lib/omnibus/packagers/msi.rb

```diff
@@ -73,9 +76,13 @@ class Packager::MSI < Packager::Base
             -cultures:en-us
             -loc "#{windows_safe_path(staging_dir, 'localization-en-us.wxl')}"
             project-files.wixobj source.wixobj
-            -out "#{windows_safe_path(Config.package_dir, package_name)}"
+            -out "#{msi_file}"
         EOH
         shellout!(light_command, returns: [0, 204])
+
+        if cert_name
```

Comment:
It's https://github.com/chef/omnibus/blob/ce9639e12b50ce5b58e63a81aa0f09f29fc95315/lib/omnibus/config.rb#L239, but I think we are moving in a different direction so I guess this is fine. I just wonder if there is a use case where a person wants to specify the cert name, but skip signing (maybe for debugging).